### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by atanasdinov**

### CODE REVIEW

File: asciidoc/edge-book/releasenotes.adoc [hunk 1]
```diff
@@ -39,34 +39,21 @@ Summary: SUSE Edge 3.1.0 is the first release in the SUSE Edge 3.1 release strea
...
 * The management cluster CAPI components are now managed via Rancher Turtles
-* Management cluster upgrades are now managed via a new Upgrade Controller
-* Edge Image Builder now supports aarch64 (tech preview)
-* Stack Validation results are now published at: https://ci.edge.suse.com
-* nm-configurator updated to utilize nmstate 2.2.36
-* Edge Image Builder Enhancements:
-** Adds support for customizing SL Micro 6.0 base images (for SLE Micro 5.5 images, EIB 1.0.x must still be used)
-** Added the ability to build aarch64 images on an aarch64 host machine (tech preview)
+* Management cluster upgrades are now managed via <<components-upgrade-controller, Upgrade Controller>>
+* Stack Validation results are now published at https://ci.edge.suse.com
```
Comment (e-minguez) [inline asciidoc/edge-book/releasenotes.adoc:43]:
```suggestion
* Stack Validation results are now published at link:https://ci.edge.suse.com[ci.edge.suse.com]
```

File: asciidoc/edge-book/releasenotes.adoc [hunk 2]
```diff
@@ -39,34 +39,21 @@ Summary: SUSE Edge 3.1.0 is the first release in the SUSE Edge 3.1 release strea
...
+** Added the ability to enable FIPS mode
 ** Added caching for container images
-** Added built image name output to build command
 ** Leftover combustion artifacts are now removed on first boot
 ** OS files and user provided certificates now maintain original permissions when copied to the final image
 ** Dependency upgrades
 *** "Phone Home" deployments are now utilizing Elemental v1.6 (upgraded from v1.4)
 *** Embedded registry is now utilizing Hauler v1.0.7 (upgraded from v1.0.1)
-*** Network customizations are now utilizing nmc v0.3.1 (upgraded from v0.3.0)
-** Image Definition Changes
```
Comment (e-minguez) [inline asciidoc/edge-book/releasenotes.adoc:62]: I think this is important enough to be here. Maybe we can have a subsection just for EIB? Something like https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#ocp-4-17-new-features-and-enhancements_release-notes

File: asciidoc/edge-book/releasenotes.adoc [hunk 3]
```diff
@@ -39,34 +39,21 @@ Summary: SUSE Edge 3.1.0 is the first release in the SUSE Edge 3.1 release strea
...
-*** The current version of the image definition has been incremented to 1.1 to include the changes below
-**** Introduced a dedicated FIPS mode option (enableFIPS) which will enable FIPS mode on the node
-**** Existing definitions using the 1.0 version of the schema will continue to work with EIB
-** Image Configuration Directory Changes
```
Comment (e-minguez) [inline asciidoc/edge-book/releasenotes.adoc:66]: Same here

File: asciidoc/edge-book/releasenotes.adoc [hunk 4]
```diff
@@ -76,11 +63,6 @@ Summary: SUSE Edge 3.1.0 is the first release in the SUSE Edge 3.1 release strea
 * Edge Image Builder Fixes
 ** Certain Helm charts fail when templated without specified API Versions: https://github.com/suse-edge/edge-image-builder/issues/481[EIB issue #481]
 ** Large Helm manifests fail to install: https://github.com/suse-edge/edge-image-builder/issues/491[EIB issue #491]
-** Fix kernelArgs issue with Leap Micro 6.0: https://github.com/suse-edge/edge-image-builder/issues/498[EIB issue #498]
```
Comment (e-minguez) [inline asciidoc/edge-book/releasenotes.adoc:79]: I think we need to put everything here

File: asciidoc/quickstart/eib.adoc [hunk 5]
```diff
@@ -15,6 +15,12 @@ Edge Image Builder (EIB) is a tool that streamlines the process of generating Cu
 
 For more information, read the <<components-eib,Edge Image Builder Introduction>>.
 
+[WARNING]
```
Comment (e-minguez) [inline asciidoc/quickstart/eib.adoc:18]: I like this better 👏🏻

File: asciidoc/edge-book/releasenotes.adoc (hunk 2 above)
Comment (atanasdinov) [reply asciidoc/edge-book/releasenotes.adoc:62]: I don't think this is relevant for product documentation. We'd want to have high level overview of the new functionality, but not technical details. Applies to all other mentions.

File: asciidoc/edge-book/releasenotes.adoc (hunk 4 above)
Comment (atanasdinov) [reply asciidoc/edge-book/releasenotes.adoc:79]: None of these are known issues in a previous version. Those came up and were part of the development work for v1.1 so they don't belong here.

File: asciidoc/edge-book/releasenotes.adoc (hunk 2 above)
Comment (e-minguez) [reply asciidoc/edge-book/releasenotes.adoc:62]: How are customers then going to find such changes? Is it documented elsewhere?

Comment (atanasdinov) [reply asciidoc/edge-book/releasenotes.adoc:62]: Yes, the [upstream](https://github.com/suse-edge/edge-image-builder/blob/main/RELEASE_NOTES.md#v110) release notes.

Comment (atanasdinov) [reply asciidoc/edge-book/releasenotes.adoc:62]: Don't get me wrong. We'd eventually want to have official page for EIB's release notes. However, such EIB specifics do not belong to the **Edge** release notes, in my opinion.

Comment (jdob) [reply asciidoc/edge-book/releasenotes.adoc:62]: IMO, for now I think we should include them. We need a better longer term approach, but I'd rather it all get mentioned than have users need to dig for it.

File: asciidoc/edge-book/releasenotes.adoc (hunk 4 above)
Comment (jdob) [reply asciidoc/edge-book/releasenotes.adoc:79]: If that's the case, I'm with Atanas. No need to distract people with bugs they never had access to.

**Review by jdob — APPROVED** [review]

File: asciidoc/edge-book/releasenotes.adoc (hunk 2 above)
Comment (e-minguez) [reply asciidoc/edge-book/releasenotes.adoc:62]: I agree with Jason on this one. We can also do something in the middle, like adding here just the 'highlights' and refer users to the upstream release notes:

```
For more details about the EIB 1.1 release and the changes, please visit the upstream release notes.
```

Or something like that.

File: asciidoc/edge-book/releasenotes.adoc (hunk 4 above)
Comment (e-minguez) [reply asciidoc/edge-book/releasenotes.adoc:79]: ACK

**Pull request merged**